DSharpPlus.UnifiedCommands/UnifiedCommandsBuilder.cs

```diff
@@ -0,0 +1,102 @@
+using System.Reflection;
+using DSharpPlus.UnifiedCommands.Message;
+using DSharpPlus.UnifiedCommands.Message.Internals;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace DSharpPlus.UnifiedCommands;
+
+public class UnifiedCommandsBuilder
+{
+    private Assembly? _assembly = null;
```

Comment:
we might want to consider multiple assemblies, so as to support loading plugins and the likes